lib/ezutils/classes/eztimestamp.php

```diff
@@ -14,6 +14,11 @@ class eZTimestamp
      \return a timestamp in UTC
     */
     public static function getUtcTimestampFromLocalTimestamp( $localTimestamp ) {
+
+        if (!$localTimestamp) {
+            return null;
```

Comment:
Legacy CS:

```
if ( !$localTimestamp )
{
```

Reply:
So:
1. spacing inside parentheses
2. blocks always on newline

Reply:
👆

Reply:
Ah, wasn't aware of that, resolved in: https://github.com/ezsystems/ezpublish-legacy/pull/1464/commits/1226e4c9df174e5effdfb582f471cd56cf6a35bc